src/core/EdFi.DataManagementService.Core/Middleware/CoreLoggingMiddleware.cs

```diff
@@ -17,12 +17,10 @@ internal class CoreLoggingMiddleware(ILogger _logger) : IPipelineStep
 {
     public async Task Execute(PipelineContext context, Func<Task> next)
     {
-        _logger.LogDebug("FrontendRequest: {FrontendRequest}", context.FrontendRequest);
```

Comment:
I remove it from this place, move it to a middleware that controls it.

Reply:
After moving this, what is the purpose of `CoreLoggingMiddleware`?

Reply:
> After moving this, what is the purpose of `CoreLoggingMiddleware`?

Respond with statusCode 500 - Unknown Error in case of any uncontrolled errors occurring in the following Core...Middleware, Now renamed to CoreExceptionLoggingMiddleware